src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -438,12 +438,11 @@ bool LibraryCallKit::inline_vector_shuffle_iota() {
 //                    ShuffleToVectorOperation<VM,Sh,E> defaultImpl)
 bool LibraryCallKit::inline_vector_shuffle_to_vector() {
   // TODO shuffle is not supported on SVE
```

Comment:
The whole patch looks good to me, except for this comment. Could you please remove it as well? Thanks!

Reply:
OK. I will remove it.